src/sysv-generator/sysv-generator.c

```diff
@@ -828,9 +827,8 @@ static int set_dependencies_from_rcnd(const LookupPaths *lp, Hashmap *all_servic
         if (r < 0)
                 return r;
 
-        STRV_FOREACH(p, sysvrcnd_path) {
-                for (i = 0; i < ELEMENTSOF(rcnd_table); i ++) {
-
+        STRV_FOREACH(p, sysvrcnd_path)
+                for (unsigned i = 0; i < ELEMENTSOF(rcnd_table); i ++) {
```

Comment:
should probably be size_t as ELEMENTOFS() reurns size_t

Reply:
It's a static table with 4 elements, `u8` would suffice ;) I don't think it's worth spinning up the CI for this...